### PR TITLE
Dwc0011/fix overgeneral exceptions warning

### PR DESCRIPTION
**Pull request by dwc0011**
Fixes issue #671 
Updated the .pylintrc to specify modules
Bumps the version
Updated the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### [0.24.6](https://github.com/plus3it/tardigrade-ci/releases/tag/0.24.6)
+
+**Released**: 2023.02.15
+
+**Summary**:
+
+* Updates .pylintrc to specify module name for overgeneral-exceptions
```
Comment (lorengordon) [inline CHANGELOG.md:13]: Needs to mention any tool versions that are included in this release. See prior release for an example. Here's the comparison link:

* https://github.com/plus3it/tardigrade-ci/compare/0.24.5...master

If you use the "Files changed" tab, it's pretty easy to see all what's updated.

**Review by lorengordon — COMMENTED** [review]
Seems fine, just needs an update to the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (dwc0011) [reply CHANGELOG.md:13]: Updated

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**